Cargo.toml

```diff
@@ -39,8 +39,9 @@ async-session  = { version = "2.0.0", optional = true }
 async-sse = "4.0.0"
 async-std = { version = "1.6.0", features = ["unstable"] }
 async-trait = "0.1.36"
-femme = "2.0.1"
+femme = { version = "2.0.1", optional = true }
 futures-util = "0.3.5"
+log = { version = "0.4.8", features = ["std"] }
```

Comment:
What's this for?

Reply:
Currently the dependency tree looks like this
```
tide
   -- kv-log-macro
      -- log (features = ["kv_unstable"])
   -- femme
      -- log (features = ["std"])
```

However, `tide` actually relies on the `std` feature in the `log` crate being turned on. Otherwise you get the following error

```
 the trait `log::kv::value::ToValue` is not implemented for `std::string::String`
```

Ideal way to solve this is actually like this: https://github.com/yoshuawuyts/kv-log-macro/pull/7